rnaget-openapi.yaml

```diff
@@ -1,704 +1,887 @@
-swagger: "2.0"
+openapi: 3.0.0
 info:
-  title: "rnaget API"
-  description: "RNA API for expression value retrieval"
-  version: "1.0.0"
-basePath: "/rnaget"
-produces:
-  - application/json
+  title: rnaget API
+  description: RNA API for expression value retrieval
```

Comment:
Should probably update the description as we now have 2 supported RNA data types

Reply:
Description updated in a9b35a6